widget-api.d.ts

```diff
@@ -209,64 +176,50 @@ interface InputProps extends Omit<TextProps, 'children' | 'width'> {
   width?: WidgetJSX.Size
   inputBehavior?: 'wrap' | 'truncate' | 'multiline'
 }
-
 interface FragmentProps extends HasChildrenProps {
   key?: BaseProps['key']
 }
-
 interface FrameProps extends BaseProps, WidgetJSX.WidgetJSXFrameProps, HasChildrenProps {}
-
 interface AutoLayoutProps extends BaseProps, WidgetJSX.WidgetJSXAutoLayoutProps, HasChildrenProps {}
-
 interface EllipseProps extends BaseProps, WidgetJSX.WidgetJSXEllipseProps {}
-
 interface RectangleProps extends BaseProps, WidgetJSX.WidgetJSXRectangleProps {}
-
 interface ImageProps extends BaseProps, WidgetJSX.WidgetJSXImageProps {}
-
 interface LineProps extends BaseProps, WidgetJSX.WidgetJSXLineProps {
   length?: WidgetJSX.Size
   strokeCap?: WidgetJSX.StrokeCap
 }
-
 interface SVGProps extends BaseProps, Partial<WidgetJSX.WidgetJSXFrameProps> {
   src: string
 }
 interface BaseProps extends WidgetJSX.WidgetJSXBaseProps {
-  // We have a custom onClick api that returns a promise
   onClick?: (event: WidgetClickEvent) => Promise<any> | void
   key?: string | number
   hoverStyle?: WidgetJSX.HoverStyle
   tooltip?: string
   positioning?: 'auto' | 'absolute'
 }
-
 interface HasChildrenProps {
   children?: FigmaDeclarativeNode | FigmaDeclarativeNode[]
 }
-
-type HexCode = string
+declare type HexCode = string
 declare namespace WidgetJSX {
-  export interface Vector {
+  interface Vector {
```

Comment:
I don't think we need to add `export` statements within the WidgetJSX namespace since we don't directly expose the WidgetJSX namespace.